libraries/AP_BoardConfig/AP_BoardConfig.h

```diff
@@ -63,6 +63,7 @@ class AP_BoardConfig {
         PX4_BOARD_PIXRACER = 4,
         PX4_BOARD_PHMINI   = 5,
         PX4_BOARD_PH2SLIM  = 6,
+        PX4_BOARD_PIXHAWK_PRO  = 7,
```

Comment:
I thought 7 was used somewhere hidden which is why AEROFC moved up to 13? Need to double-check this.